src/containers/Tenant/Diagnostics/Healthcheck/Details/Details.tsx

```diff
@@ -38,18 +36,28 @@ export const Details = (props: DetailsProps) => {
         );
     };
 
-    const renderHealthcheckIssues = () => {
+    const renderContent = () => {
+        if (error) {
+            return <div className={b('error')}>{error.statusText || i18n('no-data')}</div>;
+        }
+
+        if (!issueTrees || !issueTrees.length) {
+            return i18n('status_message.ok');
+        }
+
         return (
             <div className={b('issues-wrapper')}>
-                <IssueTree issueTree={issueTree} />
+                {issueTrees.map((issueTree) => (
+                    <IssueTree key={issueTree.id} issueTree={issueTree} />
+                ))}
             </div>
         );
     };
 
     return (
         <div className={b('details')}>
             {renderHealthcheckHeader()}
-            {renderHealthcheckIssues()}
+            {renderContent()}
         </div>
     );
```

Comment:
It will be better to wrap error and no data message into `issues-wrapper` css class to remain constant height with different content. 

Probably you could do something like this and put styles from `issues-wrapper` to `details-content-wrapper`
```
 return (
        <div className={b('details')}>
            {renderHealthcheckHeader()}
            <div className={b('details-content-wrapper')}>{renderContent()}</div>
        </div>
    );
```
Styles I'm writing about
```
&__issues-wrapper {
        overflow-x: hidden;
        overflow-y: auto;

        height: 70vh;
        max-height: 70vh;
    }
```